src/glados/TTS/phonemizer.py

```
@@ -12,16 +12,18 @@
+
 # Default OnnxRuntime is way to verbose
 ort.set_default_logger_severity(4)
 
 
 @dataclass
 class ModelConfig:
-    MODEL_NAME: Path = Path("models/TTS/phomenizer_en.onnx")
-    PHONEME_DICT_PATH: Path = Path("./models/TTS/lang_phoneme_dict.pkl")
-    TOKEN_TO_IDX_PATH: Path = Path("./models/TTS/token_to_idx.pkl")
-    IDX_TO_TOKEN_PATH: Path = Path("./models/TTS/idx_to_token.pkl")
+    MODEL_NAME: Path = resource_path("models/TTS/phomenizer_en.onnx")
```

Ruff (0.8.2)

Do not perform function call `resource_path` in dataclass defaults

(RUF009)